Common/Python/PandasConverter.cs

```diff
@@ -135,15 +135,9 @@ public PyObject GetIndicatorDataFrame(IDictionary<string, List<IndicatorDataPoin
 
                 foreach (var kvp in data)
                 {
-                    var index = new List<DateTime>();
-                    var values = new List<double>();
-
-                    foreach (var item in kvp.Value)
-                    {
-                        index.Add(item.EndTime);
-                        values.Add((double)item.Value);
-                    }
-                    pyDict.SetItem(kvp.Key.ToLowerInvariant(), _pandas.Series(values, index));
+                    var index = kvp.Value.Select(x => x.EndTime);
+                    var values = kvp.Value.Select(x => x.Value);
```

Comment:
In PandasConverter+PandasData, we have opted to use `List` instead of `Linq` expressions for performance reasons, since we would enumerate `kvp.Value` only once.

Reply:
💯 Changed it back!